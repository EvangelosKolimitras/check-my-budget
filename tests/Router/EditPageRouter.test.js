import React from "react"
import { shallow, configure } from "enzyme"
import expenses from "../Fixtures/expenses"
import toJson from "enzyme-to-json"
import { Edit } from "../../src/scripts/js/Router/EditPageRouter"
import Adapter from "enzyme-adapter-react-16"
configure({ adapter: new Adapter() })

describe("Routers", () => {
    describe("the EditPageRouter router,", () => {
        it("should render EditPageRouter correctly", () => {
            const wrapper = shallow(<Edit />)
            expect(toJson(wrapper)).toMatchSnapshot()
        })
    })
})
