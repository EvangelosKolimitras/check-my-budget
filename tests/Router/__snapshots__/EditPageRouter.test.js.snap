// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routers the EditPageRouter router, should render EditPageRouter correctly 1`] = `
<div>
  <ExpenseForm
    onSubmit={[Function]}
  />
  <button
    onRemove={[Function]}
  >
     Remove Item 
  </button>
</div>
`;
